### PR TITLE
Firebase Auth and App Engine Python API

### PR DESCRIPTION
**Pull request by archelogos**
A quick tutorial that shows how to use JWT from Firebase Auth in a Python application made with Falcon.

It is based on some concepts explained in the previous tutorial: https://cloud.google.com/community/tutorials/appengine-python-falcon

### CODE REVIEW

**Comment by archelogos** [comment]
It seems a good idea to create a new branch for each tutorial...

File: tutorials/appengine-python-api-firebase-auth/app.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+runtime: python27
+api_version: 1
+threadsafe: true
+
+handlers:
+  - url: /.*
+    script: api.app
+
+env_variables:
+  GCLOUD_PROJECT: 'clear-beacon-145712'
```
Comment (jmdobry) [inline tutorials/appengine-python-api-firebase-auth/app.yaml]: s/clear-beacon-145712/YOUR_PROJECT_ID/

but actually, doesn't App Engine Standard set the GCLOUD_PROJECT env var for you? I know it does in App Engine Flexible Environment

Comment (archelogos) [inline tutorials/appengine-python-api-firebase-auth/app.yaml]: As long as I know that variable is not set using the dev_appserver script in development mode and the firebase-admin library needs it.

**Comment by jmdobry** [comment]
@JimTravis I think the CLA is mad because you added your own commit to the PR via GitHub's interface, so it used your noreploy.github.com email, which does not have a CLA. The CLA check was passing before your first commit. This PR is LGTM.

**Comment by archelogos** [comment]
🎉🎉

**Pull request merged**